addons/mail/static/src/components/activity_mark_done_popover/activity_mark_done_popover_tests.js

```diff
@@ -234,10 +244,50 @@ QUnit.test('activity mark done popover mark done and schedule next', async funct
     let feedbackTextarea = document.querySelector('.o_ActivityMarkDonePopover_feedback');
     feedbackTextarea.focus();
     document.execCommand('insertText', false, 'This task is done');
-    document.querySelector('.o_ActivityMarkDonePopover_doneScheduleNextButton').click();
+    await afterNextRender(() => {
+        document.querySelector('.o_ActivityMarkDonePopover_doneScheduleNextButton').click();
+    });
     assert.verifySteps(
         ['action_feedback_schedule_next'],
-        "Mark done and schedule next button should call the right rpc"
+        "Mark done and schedule next button should call the right rpc and not trigger an action"
```

Comment:
The "and not trigger an action" part only makes sense as `verifySteps` comment if there is actually a step inside inside the action, as I suggested in another comment.